src/quicr_client.cpp

```diff
@@ -81,40 +97,45 @@ class QuicRTransportDelegate : public ITransport::TransportDelegate
   {
   }
 
-  virtual void on_new_media_stream(
+  virtual void on_new_stream(
     const qtransport::TransportContextId& /* context_id */,
-    const qtransport::MediaStreamId& /* mStreamId */)
+    const qtransport::StreamId& /* mStreamId */)
   {
   }
 
   virtual void on_recv_notify(const qtransport::TransportContextId& context_id,
-                              const qtransport::MediaStreamId& mStreamId)
+                              const qtransport::StreamId& streamId)
   {
-    // TODO: Consider running the below async or in a thread
+    for (int i=0; i < 60; i++) {
```

Comment:
Noob question: Why 60?

Reply:
Just a starting point. We don’t want to block the event loop thread, so this does a little at a time. We’ll need to refactor this based on either config or breaking this out into threads or something else.